README.adoc

```diff
@@ -32,6 +33,8 @@ This Docker image provides:
 * Source highlighting using http://rouge.jneen.net[Rouge], https://rubygems.org/gems/coderay[CodeRay] or https://pygments.org/[Pygments]
 * https://github.com/asciidoctor/asciidoctor-confluence[Asciidoctor Confluence] {ASCIIDOCTOR_CONFLUENCE_VERSION}
 * https://github.com/asciidoctor/asciidoctor-bibtex[Asciidoctor Bibtex] {ASCIIDOCTOR_BIBTEX_VERSION}
+* https://github.com/Mogztter/asciidoctor-kroki[Asciidoctor Kroki] {ASCIIDOCTOR_KROKI_VERSION}
```

Comment:
Can you transplant this change in `README-original.adoc`?
The file `README.adoc` is generated by the command `make README` by templating `README-original.adoc` with the content of `tests/env_vars.yml`. 
This "not-easy" behavior will be fixed soon, but if you don't transplant in the PR, the next PR will lost your changes.